webcompat/api/endpoints.py

```diff
@@ -181,7 +175,7 @@ def get_category_from_search(issue_category):
     that maps to a label. This uses the Issues API, which is less costly than
     the Search API.
     '''
-    category_list = ['contactready', 'needscontact',
+    category_list = ['needstriage', 'contactready', 'needscontact',
```

Comment:
nit: Can we have `needstriage` after `needsdiagnosis` so the list remains in alphabetical order?